src/VisualStudio/VisualBasic/Impl/ProjectSystemShim/EntryPointFinder.vb

```diff
@@ -15,14 +16,15 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.ProjectSystemShim
             Me._findFormsOnly = findFormsOnly
```

Comment:
i'm not really understanding the purpose of this flag.

Reply:
the Visual Basic project system has a feature that allows you set any Form as the entry point. This was to enable more rapid development for winforms developers. The entry point service only discovers winforms types in a compilation when in this mode.